(window.webpackJsonp=window.webpackJsonp||[]).push([[13],{"02xs":function(e,t){e.exports=[{name:"dyspyosis",description:"Python package to detect dysbiosis in microbiome data",year:2024,URLs:[{name:"github",link:"https://github.com/raeslab/dyspyosis"}]},{name:"RLdbRDA",description:"RaesLab distance-based redundancy analysis",year:2024,URLs:[{name:"github",link:"https://github.com/raeslab/RLdbRDA"}]},{name:"loreplotr",description:"Logistic Regression Plots for R",year:2024,URLs:[{name:"github",link:"https://github.com/raeslab/loreplotr"}]},{name:"lorepy",description:"Logistic Regression Plots for Python",year:2023,URLs:[{name:"github",link:"https://github.com/raeslab/lorepy"},{name:"Zenodo",link:"https://zenodo.org/record/8324902"}]},{name:"statannotations",description:"Python package to add annotations to seaborn plots",year:2023,URLs:[{name:"github",link:"https://github.com/trevismd/statannotations"},{name:"Zenodo",link:"https://zenodo.org/record/8396665"}]},{name:"CoNekT",description:"Co-Expression Network Toolkit",year:2018,URLs:[{name:"github",link:"https://github.com/sepro/CoNekT"}]},{name:"PhytoNet",description:"Comparative co-expression network analyses across phytoplankton and land plants",year:2018,URLs:[{name:"link",link:"http://www.gene2function.de"}]},{name:"LSTrAP",description:"Large-Scale Transcriptomics Analysis Pipeline",year:2017,URLs:[{name:"github",link:"https://github.com/sepro/LSTrAP"}]},{name:"Plasmid DB",description:"Lab Management software",year:2015,URLs:[{name:"link",link:"http://sebastian.proost.science/plasmid-db/"}]},{name:"PLAZA",description:"Online comparative genomics platform",year:2014,URLs:[{name:"link",link:"http://bioinformatics.psb.ugent.be/plaza/"}]},{name:"Trapid",description:"Online transcript annotation and phylogeny tool",year:2013,URLs:[{name:"link",link:"http://bioinformatics.psb.ugent.be/webtools/trapid/"}]},{name:"i-ADHoRe",description:"Software to detect genomic homology",year:2012,URLs:[{name:"github",link:"https://github.com/VIB-PSB/i-ADHoRe"}]}]},WFGw:function(e,t,n){"use strict";n.r(t);var a=n("q1tI"),i=n.n(a),o=n("Bl7J"),s=n("qJ7i"),r=function(e){var t=e.badges.map((function(t,n){return i.a.createElement(s.a,{key:e.key+"_"+n,link:t.link},t.name)}));return i.a.createElement("table",{className:"item-table"},i.a.createElement("tbody",null,i.a.createElement("tr",null,i.a.createElement("td",{className:"item-name"},e.name||""," ",t),i.a.createElement("td",{className:"item-when"},e.when||"")),i.a.createElement("tr",null,i.a.createElement("td",{className:"item-org",colSpan:"2"},e.description||""))))},l=n("02xs"),c=n.n(l);t.default=function(e){var t=c.a.map((function(e,t){return i.a.createElement(r,{key:t,name:e.name,when:e.year,description:e.description,badges:e.URLs})}));return i.a.createElement(o.a,{pageTitle:"Software",pageDescription:"Learn more about things I developped",showTitle:!0},t)}},qJ7i:function(e,t,n){"use strict";var a=n("q1tI"),i=n.n(a);t.a=function(e){return i.a.createElement("a",{href:e.link,className:"badge"},e.children)}}}]);
//# sourceMappingURL=component---src-pages-software-js-52133683ce50c1583bbf.js.map